### PR TITLE
 fix(setAverageIPScalarRange)

### LINKED ISSUE #1924: AverageIPScalar not working
### High-level description
AverageIPScalar not behaving as expected.

I've been trying to use the AverageIPScalar so limit the values rendered with and average blend mode. I haven't had any luck getting this running. 

### Steps to reproduce
In the VolumeMapperBlendModes example add:

`mapper.setAverageIPScalarRange(0.0, 0.9);`

The volume is now not visible when selecting average blend mode.

### Expected behavior
The volume would average all the values between 0.0 and 0.9

### PR DESCRIPTION
**Pull request by GeorgeFuller**
### Context
fix #1924
https://discourse.vtk.org/t/averageipscalarrange/5544/4

### Changes
averageIPScalarRange working for single component volumes (from some angles).
averageIPScalarRange can now accept float values.

### Results
Previously setting the range to a float: 
`mapper.setAverageIPScalarRange(0.0, 0.9);`
would result in a blank canvas on a volume and shader errors.

Currently: Adjusting the max slider between 0.0 and 1.0 results in this: 
averageIPScalarTest

### Testing
run 
`npm run example -- VolumeMapperBlendModes`
switch the drop down to Average Intensity.

### CODE REVIEW

File: Sources/Rendering/OpenGL/VolumeMapper/index.js [hunk 1]
```diff
@@ -232,18 +232,31 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
...
+    if (Number.isInteger(averageIPScalarRange[0])) {
+      min = `${averageIPScalarRange[0]}.0`;
+    } else {
+      min = `${averageIPScalarRange[0]}`;
+    }
+    if (Number.isInteger(averageIPScalarRange[1])) {
+      max = `${averageIPScalarRange[1]}.0`;
+    } else {
+      max = `${averageIPScalarRange[1]}`;
+    }
```
Comment (daker) [inline Sources/Rendering/OpenGL/VolumeMapper/index.js:249]: Why not use [toFixed](https://caniuse.com/?search=toFixed)() instead?

something like this 

```js
const a = averageIPScalarRange[0];
const b = averageIPScalarRange[1];

min = (Number.isInteger(a)) ? a.toFixed(1).toString() : (a.toString());
max = (Number.isInteger(b)) ? b.toFixed(1).toString() : (b.toString());
```
ex : 
```js
let i = 1
(Number.isInteger(i)) ? i.toFixed(1).toString() : (i.toString());
"1.0"
i = 1.25
(Number.isInteger(i)) ? i.toFixed(1).toString() : (i.toString());
"1.25"
```

Comment (GeorgeFuller) [reply Sources/Rendering/OpenGL/VolumeMapper/index.js:249]: Oh I like that, looks clean and works well! I changed it to this. Never used toFixed before

**Comment by GeorgeFuller** [comment]
I added a couple of changes to the volume shader which now has this behaving as I'd expect it to.

**Review by martinken — APPROVED** [review]
LGTM

**Comment by martinken** [comment]
Any reason not to merge this?

**Comment by martinken** [comment]
But other than that any reason not to merge?

**Comment by jourdain** [comment]
I don't think so..

**Pull request merged**